README.md

```diff
@@ -43,8 +43,20 @@ The 2nd param is optional, and correspond to the theme to use. Light or Dark. By
      * @return Doordeck the current instance of the SDK
      */
     fun initialize(apiKey: String, darkMode: Boolean = true): Doordeck
+``` 
+
+#### Tweak the NFC Uri settings
+Default values for a NFC Uri link would be `https://doordeck.link/uuid/${uuid}`.
```

Comment:
```suggestion
Default values for a NFC Uri link would be `https://doordeck.link/${uuid}`.
```

`uuid` part should have never been there

Reply:
I would have to change and test the NFC project first. Bear with me.

Reply:
Revised and fixed